src/lib/permutations/keccak_fips/keccak_fips.h

```diff
@@ -0,0 +1,127 @@
+/*
+* Keccak-FIPS
+* (C) 2010,2016 Jack Lloyd
+* (C) 2023 Falko Strenzke
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_KECCAK_FIPS_H_
+#define BOTAN_KECCAK_FIPS_H_
+
+#include <botan/hash.h>
```

Comment:
Please remove this superfluous include. It prevents a successful build with ` ./configure.py --minimized-build --enable-modules=keccak_fips`.

```suggestion
```

Reply:
Thanks for the corrections